content/md/assets/upload-assets.md

```diff
@@ -13,6 +13,10 @@ Assets mass upload is pretty convenient especially if you want to update your as
 
 The PIM uses the filename to know for which asset the file have to be used and if this asset is localized or not.
```

Comment:
C'est vrai que c'est pas très précis ce qui avait été mis.
Après la 1ère phrase, je mettrai "To check if the asset already exists, the PIM compares the existing assets codes with the filename, whose characters are not allowed in an asset code (only letters, numbers and underscores) are replaced by an underscore."